spring-kafka-test/src/test/java/org/springframework/kafka/test/extensions/AddressableEmbeddedBrokerTests.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.kafka.test.rule;
+package org.springframework.kafka.test.extensions;
```

Comment:
Doesn't look like this class logic does something with log adjustment.
Might be better to move it to some other package.
Probably to the root one alongside with the `EmbeddedKafkaKraftBrokerTests`